src/main/java/net/sf/jabref/importer/EntryFromPDFCreator.java

```diff
@@ -64,7 +64,7 @@ public boolean accept(File f) {
         PdfImporter pi = new PdfImporter(JabRefGUI.getMainFrame(), JabRefGUI.getMainFrame().getCurrentBasePanel(), JabRefGUI.getMainFrame().getCurrentBasePanel().getMainTable(), -1);
```

Comment:
It would be nice, if you could at least assign the getMainFrame value to a local variable. Would improve the readability a bit.